social_core/backends/orcid.py

```diff
@@ -70,11 +70,11 @@ def get_user_details(self, response):
         if person:
             name = person.get('name')
 
-            fullname = name
-
             if name:
                 first_name = name.get('given-names', {}).get('value', '')
                 last_name = name.get('family-name', {}).get('value', '')
+                fullname = first_name + ' ' + last_name
+                fullname = fullname.strip()
```

Comment:
Can you please look why this is not covered by tests? The user data payload seems to be present in the test case, but it doesn't seem to be used.